test/local/main.cpp

```diff
@@ -89,8 +89,6 @@ int main(int argc, char** argv) {
 
     std::cout << "SQLite Version: " << SQLITE_VERSION << std::endl;
 
-    std::cout << "Test Error Message: " << TP::sqlite::getErrorString(SQLITE_ABORT) << std::endl;
```

Comment:
Removed because the equivalent would be testing sqlite3_errstr, a function we don't own.